vdb/recon/sniper.py

```diff
@@ -5,16 +5,24 @@
 import envi.memory as e_mem
 import vtrace.breakpoints as vt_breakpoints
 
-def getStackArg(trace, argidx):
+def getArg(trace, argidx):
     '''
     Assuming we are at the instruction after
-    a call, grab the stack argument at the specified
+    a call, grab the argument at the specified
     index (skipping the saved instruction pointer).
     '''
-    stack = trace.getStackCounter()
+    cc = trace.getEmulator().getCallingConvention("stdcall")
+
+    if "64" in trace.getMeta("Architecture"):
```

Comment:
another instance of "fuzzy" substring comparison.  on amd64, the "Architecture" meta will always be exactly "amd64" so lets test for that.